src/main/java/com/rtb/projectmanagementtool/project/ProjectController.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * ProjectController.java - this file implements ProjectController, which controls the logic of
+ * working with Project objects.
+ */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.HashSet;
+
+public class ProjectController {
+  private DatastoreService datastore;
+  private HashSet<ProjectData> projects;
```

Comment:
This may be ok.  But also think about what you'd do with this list of projects later on.  If you'll be passing this around to other consumers (controllers or servlets), it might be best to not own an instance here and instead just pass the container in when doing load/create/remove operations.  This way you wouldn't have to worry about the data being copied when passing between consumers.